### ISSUE TITLE
Drag problem in Facebook layout type.

### ISSUE THREAD

**Issue by mimmo00** [samvermette/SVSegmentedControl#50]
Hi, first of all, congratulations for the wonderful uicontrol.
I've used several times your switch, no problem. But now I have implemented it in a  Facebook layout type, a draggable view. When move the thumb, arrives in the middle and come back. With a single touch does work.

You know how to solve this problem?

Thanks man

**Comment by samvermette** [comment]
It's hard to visualize the problem you're describing. Can you post a video describing the issue? That would help.

**Comment by mimmo00** [comment]
Hi, sure. Sorry for the mess inside the view: P

http://d.pr/v/7DVM

**Comment by samvermette** [comment]
Nice implementation of SVSegmentedControl...

It looks like the touch is getting cancelled somehow. Don't really have the time to recreate your setup, but here are a few clues:
- Have a look at the Tracking in SVSegmentedControl.m
- Add breakpoints to see what's going on.
- Another thought is that the segmented control drag gets cancelled by the facebook layout type view. Just like a UITableView will cancel the touch of a cell when you drag it by enough pixels. Perhaps there's a similar property, like `canCancelContentTouches` on the facebook layout class?

**Comment by mimmo00** [comment]
Thanks, tried to experiment and let you know.
I have only one doubt, I do not think the problem is in the layout, because I have other uicontrol inside, and do not have this problem.

**Comment by samvermette** [comment]
You're right, if there's no problem with your UISwitch then there shouldn't be a problem with SVSegmentedControl. It's probably something in one of the tracking methods then...
